static/js/wagtail_lazy_load.js

```diff
@@ -0,0 +1,25 @@
+/*eslint-env jquery*/
+/*eslint semi: ["error", "always"]*/
+
+export default function wagtailLazyLoader() {
+  const observerConfig = {
+    root:       null,
+    rootMargin: "0px",
+    threshold:  0
+  };
+
+  const imgs = document.querySelectorAll(".wagtail-lazy-load");
```

Comment:
Selector can be changed to `img.lazy` if you change that class name

Reply:
updated.